tests/inputFields.spec.ts

```diff
@@ -53,7 +32,10 @@ test('Update pet type', async ({page}) => {
 
   /* 6. Add the assertion that the first pet type in the list of types has a value "rabbit" */
   // Expecting the newly updated entry to say "rabbit"
-  expect(await page.locator('input[name=pettype_name]').first().inputValue()).toEqual('rabbit')
+  //expect(await page.locator('input[name=pettype_name]').first().inputValue()).toEqual('rabbit')
+ // await expect(page.locator('input[name=pettype_name]'))
```

Comment:
Keep your code clean.
If something is not needed, in the form of old commented code, it must be deleted to keep everything nicely organized.